compatibility/compatibility-vec-common/src/test/resources/vec12x/vec121-test.vec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would have expected that file to be moved, not duplicated. Same for the other files.